### DISCUSSION TITLE
Can't install default theme

### DISCUSSION THREAD

**Discussion by aphile** [atom/atom#22199] [Q&A]
Hello. I am brand new to Atom, and I am having issues applying the default themes. When I go the Settings > Themes the atom-dark-sytax theme is not available in the drop down even though when I click on it, it says it is pre-installed by default. 
I am on Atom v. 1.56.0, Mac OS v. 11.2.3.
![Screen Shot 2021-04-14 at 1 34 32 PM](https://user-images.githubusercontent.com/52717927/114754110-32b56980-9d26-11eb-8a51-ef9c13dd0b80.png)

I am not sure if this is related, but I have not installed the "git" developer tools yet. i just want to do simple HTML/CSS, so I don't know if I need them or not. When I started to install them, it said it would take 1 hr 39 min on my high-speed internet, so I cancelled the download. 

Any help is appreciated.

**Comment by UziTech** [comment]
In your screen shot "Atom Dark" is already selected for your syntax theme.

**Reply by aphile** [reply]
Oh, I did not notice that, but when I look at the file the text is all white. It does not have the different colors for different code elements that is shown in the preview for that theme, which is what I am really going for here.

**Reply by aphile** [reply]
I just found the issue. I had copied a section of code from another page. When I added the proper html boiler plate, the colors stared to show up. I knew it had to be something simple I was missing!

**Reply by shats15** [reply]
Grouppromasters
